docs_src/getting-started/Ch-GettingStartedSDK-C.md

```diff
@@ -1,213 +1,210 @@
-**localhost**. To do that, modify the configuration.toml file so that
-the top section looks like this:
-
-``` toml
-[Service]
-Host = "172.17.0.1"
-Port = 49992
-```
+Now update the configuration for the new device service.    This documentation provides a new configuration.toml file.  This configuration file:
+- changes the port the service operates on so as not to conflict with other device services
+- alters the the auto event frequency, which determines when the device service collects data from the simulated device (every 10 seconds)
+- sets up the initial provisioning of the random number generating device when the service starts
+
+Download  **[configuration.toml](configuration.toml)** and save the file to the ./res folder.
 
 ## Rebuild your Device Service
 
-Now you have your new Device Service, modified to return a random
-number, a Device Profile that will tell EdgeX how to read that random
-number, as well as a configuration file that will let your Device
-Service register itself and it's Device Profile with EdgeX, and begin
+Now you have your new device service, modified to return a random
+number, a device profile that will tell EdgeX how to read that random
+number, as well as a configuration file that will let your device
+service register itself and it's device profile with EdgeX, and begin
 taking readings every 10 seconds.
 
-1.  Rebuild your Device Service to reflect the changes that you have
-    made:
+Rebuild your Device Service to reflect the changes that you have made:
 
-        gcc -I$CSDK_DIR/include -L$CSDK_DIR/lib -o device-example-c template.c -lcsdk
+``` bash
+gcc -I$CSDK_DIR/include -L$CSDK_DIR/lib -o device-example-c template.c -lcsdk
+```
 
 ## Run your Device Service
 
 Allow your newly created Device Service, which was formed out of the
 Device Service C SDK, to create sensor mimicking data which it then
 sends to EdgeX.
 
-1.  Follow the [Getting Started Users](./Ch-GettingStartedUsers.md)
-    guide to start all of the EdgeX services in Docker. From the folder
-    containing the docker-compose file, start EdgeX with a call to:
+1.  Follow the [Getting Started with Docker](./Ch-GettingStartedUsers.md) guide to start all of EdgeX. From
+    the folder containing the docker-compose file, start EdgeX with the
+    following call:
 
-        docker-compose up -d
+    ``` bash
+    docker-compose up -d
+    ```
 
-2.  Back in your custom Device Service directory, tell your device
+2.  Back in your custom device service directory, tell your device
     service where to find the `libcsdk.so`:
 
-        export LD_LIBRARY_PATH=$CSDK_DIR/lib
+    ``` bash
+    export LD_LIBRARY_PATH=$CSDK_DIR/lib
+    ```
 
 3.  Run your device service:
 
-        ./device-example-c
+    ``` bash
+    ./device-example-c
+    ```
 
-4.  You should now see your Device Service having it's /Random command
+4.  You should now see your device service having it's /Random command
```

Comment:
*its

Reply:
is this supposed to be "/Random"? or just "Random"?